pandas/tests/tseries/test_frequencies.py

```diff
@@ -305,6 +310,10 @@ def test_period_str_to_code():
     assert (frequencies._period_str_to_code('A') == 1000)
     assert (frequencies._period_str_to_code('A-DEC') == 1000)
     assert (frequencies._period_str_to_code('A-JAN') == 1001)
+    assert (frequencies._period_str_to_code('Y') == 1000)
```

Comment:
Not sure what you mean here: neither `YS` nor `AS` are considered valid "period strings"